.github/workflows/ci.yml

```diff
@@ -164,6 +188,7 @@ jobs:
           args: --release --target=riscv64gc-unknown-none-elf --manifest-path ${{ matrix.crates }}/Cargo.toml
 
   format:
+    name: Formatting
```

Comment:
`Formatting` looks like it's going to actually change the format...

Reply:
```suggestion
    name: Check Formatting
```